src/api/namespaces/atomicmarket/handlers/sales.get-sales.test.ts

```diff
@@ -177,42 +177,28 @@ describe('AtomicMarket Sales API', () => {
                 .to.deep.equal([sale_id]);
         });
 
-        txit('throws error when maximum price filter is set without settlement symbol', async () => {
+        txit('filters by minimum price', async () => {
             let err;
             try {
```

Comment:
I usually use for this a plugin in chai, 

https://www.npmjs.com/package/chai-as-promised

Reply:
I would have used that if it had been available - I'm definitely for using that